packages/checkbox/src/index.ts

```diff
@@ -141,6 +144,7 @@ function normalizeChoices<Value>(
       value: choice.value,
       name,
       short: choice.short ?? name,
+      checkedName: choice.checkedName ?? undefined,
```

Comment:
```suggestion
      checkedName: choice.checkedName ?? name,
```

So that we can make this key always defined in the normalized type, reducing amount of conditional we need elsewhere.